app/models/form/lettings/questions/first_time_property_let_as_social_housing.rb

```diff
@@ -7,12 +7,22 @@ def initialize(id, hsh, page)
     @type = "radio"
     @check_answers_card_number = 0
     @hint_text = ""
-    @answer_options = ANSWER_OPTIONS
+    @answer_options = answer_options
```

Comment:
I think you should be able to remove this line

Reply:
nice, you're right